utils_cv/tracking/plot.py

```diff
@@ -0,0 +1,156 @@
+from collections import OrderedDict
+import os.path as osp
+from typing import Dict, List, Tuple
+import cv2
+import numpy as np
+import pandas as pd
+
+from .bbox import TrackingBbox
+
+
+def plot_results(
+    results: Dict[int, List[TrackingBbox]],
+    video_path: str,
+    frame_rate: int = 30,
```

Comment:
Should the frame rate here simply be the same frame rate as the video? Maybe there is something like this:
frame_rate = int(video.get(cv2.FRAME_RATE))